tests/unit/components/details.js

```diff
@@ -0,0 +1,65 @@
+var sd = require('skin-deep'),
+    expect = require('chai').expect,
+    Details = require('../../../ui/components/details.jsx'),
+    passDetails = require('../../fixtures/components/pass-details.js'),
+    failDetails = require('../../fixtures/components/fail-details.js');
+
+describe('Details', function() {
+  var tree;
+
+  it('should render the baseline', function() {
+    tree = sd.shallowRender(<Details paths={passDetails}/>);
+
+    expect(tree.findNode('.baseline')).to.not.be.false;
+  });
+
+  describe('Pass details', function() {
+    it('should not render the diff', function() {
+      tree = sd.shallowRender(<Details paths={passDetails}/>);
```

Comment:
This is duplicated code and should sit in a `beforeEach` block. You should have a `describe` block that contains all the tests that are suitable for passed and failed tests + the ones that apply only to pass tests. Then, in another `describe` block, place the tests that apply only to failed tests.